### PR TITLE
Share Samples base branch with Development

### PR DESCRIPTION
**Pull request by joshsadam**
## Description of changes

Complete refactor of how to share / move samples between projects.

## Related issue
#1005 in that it removed one of the menu options from the project samples page

## Checklist

Things for the developer to confirm they've done before the PR should be accepted:

*   [X] CHANGELOG.md (and UPGRADING.md if necessary) updated with information for new change.
*   [X] Tests added (or description of how to test) for any new features.
*   [X] User documentation updated for UI or technical changes.

### CODE REVIEW

File: src/main/resources/i18n/messages.properties [hunk 1]
```diff
@@ -341,7 +341,44 @@ project.members.add.error=An error occurred while adding the user to the project
...
+# ========================================================================================== #
+# ShareSamples Component                                                                     #
+# ========================================================================================== #
+ShareNoSamples.description=You have no samples selected to share, please go back to the project samples page and select some samples
+ShareNoSamples.link=Go to Project Samples Page
+ShareSamples.title=Share Samples with Another Project
+ShareSamples.projects=Select a project to share samples with
+ShareSamples.ready=These samples are ready to copy
+ShareSamples.exists=Samples that exist in the target project and will not be copied
+ShareSamples.avatar.unlocked=This same is modifiable in this project.
```
Comment (deepsidhu85) [inline src/main/resources/i18n/messages.properties:353]: Change `same` to `sample` on this line and the one below it

File: src/main/resources/i18n/messages.properties [hunk 2]
```diff
@@ -341,7 +341,44 @@ project.members.add.error=An error occurred while adding the user to the project
...
+ShareSamples.no-samples.message=All samples exist in the target project
+ShareSamples.no-samples.description=Since these samples exists, there is no reason to re-share them.
+ShareSamples.some-samples.message={0} samples exist in the target project and will not be re-shared.
+ShareSamples.checkbox.remove=Remove samples from current project (move samples)
+ShareSamples.checkbox.lock=Prevent modification of samples in target project (only when copying samples)
+ShareButton.button=Share Samples
+server.ShareSamples.move-error=There was an error moving these samples to {0}.  If the problem persists, please contact a system administrator.
+server.ShareSamples.copy-error=There was an error copying these samples to {0}.  If the problem persists, please contact a system administrator.
+ShareError.reset=Start Over
+ShareError.return=Return to Project's Samples
```
Comment (deepsidhu85) [inline src/main/resources/i18n/messages.properties:366]: Change to `Return to Project Samples page`

**Review by deepsidhu85 — CHANGES_REQUESTED** [review]
A few changes for now

File: CHANGELOG.md [hunk 3]
```diff
@@ -6,6 +6,7 @@ Changes
 * [REST]: Fixed bug where analysis output file bytes were being included in responses when only JSON should have been sent. (21.09.1)
 * [REST]: Fixed bug prevending REST API clients from updating the `collectedDate` on samples.
 * [Developer]: Updated `antd` to version 4.16.13
+* [UI]: Recreated the share / move samples between projects. Now uses an independent page.
```
[the remark this answers is not in the document]
Comment (joshsadam) [reply CHANGELOG.md]: Updated in 3f7a782

File: src/main/resources/i18n/messages.properties (hunk 1 above)
Comment (joshsadam) [reply src/main/resources/i18n/messages.properties]: Fixed in 32d95d2

File: src/main/resources/i18n/messages.properties (hunk 2 above)
Comment (joshsadam) [reply src/main/resources/i18n/messages.properties]: Updatd in 3598392

**Review by deepsidhu85 — APPROVED** [review]

**Pull request merged**